detail.py

```diff
@@ -0,0 +1,24 @@
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+
+@view_function
+def process_request(request, product: cmod.Product):
+
+    for index, i in enumerate(request.last_five):
```

Comment:
easiest way to do this is just loop through request.last_five like this:
for i in request.last_five:

Reply:
That works, but the 'index' and 'enumerate' in the forloop allow me to get the index of the item that i pop() later on.